tests/integration/backend.rs

```diff
@@ -36,6 +36,34 @@ macro_rules! make_quickcheck {
             }
         }
     };
+    // Similar to the previous branch, but adds a PRNG to method invocation as a last argument
+    (#[with_rng] $method:ident ( self: $self_ty:ty, $($arg:ident: $t:ty),* $(,)? ) ) => {
+        quickcheck::quickcheck! {
+            fn $method(nbi: $self_ty, $($arg: $t),*, rng: Prng) -> bool {
```

Comment:
Why does this require a special case? Isn't Prng a normal argument that you could have just passed to the function?